process/how_to_suggest_new_changes.md

```diff
@@ -0,0 +1,26 @@
+## How to Suggest New Code, Documentation, or Other Development Artifacts
+
+#### Context:
+
+Our project management system is driven by issues, which means that it tracks tasks through issues rather than pull requests. To propose a new activity, such as reporting a bug or making changes to the repository, you need to create an issue with a detailed description of what changes you want to make and why they are important. The issue will be discussed at a developer meeting, and if it is deemed necessary, it will be added to a sprint, and a developer will be assigned to work on it.
+
+However, there are cases when you have already made changes to the code or documentation that address the issue you want to create. For example, you may have conducted initiative research, tested your ideas, experimented with the code, made many modifications, and then found a significant improvement that makes the project much faster. Alternatively, while working on another task, you may have discovered and fixed a bug. In these cases, you already have a set of modifications that you want to suggest adding to the repository.
```

Comment:
It seems like making something like this an official process may enable contributors to work too much on things that may not have been deemed a priority by the team. We need to discuss this with the broader team but I am less supportive of this method than previous discussions about how to manage PRs without issues assigned in the future.

Reply:
I do not believe _every_ PR requires an issue. For example, misspelling or minor grammatical changes in code comments can be easily fixed and reviewed without needing the debate that this procedure outlines; however, there is a lot of value in communicating to outside contributors what the process for incorporating their changes into the product is.